### DISCUSSION TITLE
Loco Node Position System Configuration

### DISCUSSION THREAD

**Discussion by danielcmdeveloper** [bitcraze/discussions#1521] [Q&A] [answered]
We are trying to configure the Loco Position Nodes positioning system and we have some doubts.

Doing a first test, we have connected 2 Loco Position Node to see if from CFClient could be configured, but we didn't get visibility.
We would like to know if we should connect 8 or 6 Loco Position Nodes so that from CFClient, connected to a CrazyFlie we can see all the positioning beacons.
We have followed the steps shown on the CrazyFlie website but there is something we think we are not doing right.

The Loco Position Nodes are updated and the drone is at a close distance to them.

Is there any configuration that needs to be done inside CFClient?

Thank you very much.

**Comment by knmcguire** [comment]
Hi! 

So currently it could be that the positioning nodes are on a different method than the crazyflie is. That is necessary for them to be able to connect to the crazyflie and send the info through the crazyflie to the cfclient.

Are the LEDs on of the loco positioning nodes? Have you tried selecting different positioning techniques for the one crazyflie (TWR, TDOA2, TDOA3)?

**Comment by danielcmdeveloper** [comment]
Thank you very much for the recommendations.
We will review what you told us and verify all the configuration parameters you have told us.
We will give you feedback on the result shortly.
Thank you very much for the help.

**Comment by danielcmdeveloper** [comment]
Hello again,
We have tested and still have the same problem.
With two positioning beacons, we only get a response from one of them, i.e. with both of them on and with the LEDs on, we only get recognition from one of them. 
We have tried changing the detection system to all modes and the same thing happens.
Attached is an image of what CFCliente shows.
We have been able to observe that one of the positioning beacons may be configured in TDOA2 and the other in TDOA3.
Is it possible to change the mode of each of the beacons?

Thank you very much for everything.
Best regards
![LocoNodePositionProblem](https://github.com/user-attachments/assets/ea83786a-a8ac-4844-9dc7-e9dcf7d6ad4c)

**Reply by knmcguire** [reply]
Ah oke that can be it. The crazyflie can not communicate with the nodes set in different modes, only 1 mode at a time. So in this case, anchor/node 0 is configured to TWR, but you can configure it with change mode to any of the others with 'Change mode'. If you set the Crazyflie to TDOA2 or TDOA3, and connect to any of the others, you can change the mode in the same way as well. These are specific instructions on how to do that: https://www.bitcraze.io/documentation/tutorials/getting-started-with-loco-positioning-system/#switching-system-mode-to-tdoa

But you can also manually change the anchors mode with the tool shown in the tutorial. https://www.bitcraze.io/documentation/tutorials/getting-started-with-loco-positioning-system/#update-the-nodes. Make sure that all nodes have a unique ID

**Comment by danielcmdeveloper** [comment, marked as answer]
Thank you very much for your answers.

We have managed to solve the problem and the positioning beacons are now identified. Indeed it was that they were not in the same positioning mode and that they all had the same ID.

Thank you very much for your help.
![LoPS](https://github.com/user-attachments/assets/9110df1f-ffd3-4a27-bbda-62814f9bbb1c)

**Reply by knmcguire** [reply]
Got to hear! If you have any other issue don't hesitate to start a new thread

**Reply by danielcmdeveloper** [reply]
Thank you very much for everything. :-)
